homeworks/david.tereshchenko_DavidTereshchenko/GameProject/successful.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="css/successful.css">
+        <title>Game Project</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <div class="header__container">
+                    <div class="header__logo">
+                        <img src="img/logo.png" alt="logo" width="55" height="58">
```

Comment:
It would be better to use image icons in SVG format and images in WebP, JPG, or JPEG formats, as PNG files tend to have larger file sizes compared to other image formats